t/t-resume-http-range.sh

```diff
@@ -85,3 +85,42 @@ begin_test "resume-http-range-fallback"
 )
 end_test
 
+begin_test "resume-http-range-retry"
+(
+  set -e
+
+  reponame="resume-http-range-retry"
+  setup_remote_repo "$reponame"
+
+  clone_repo "$reponame" $reponame
+
+  git lfs track "*.dat" 2>&1 | tee track.log
+  grep "Tracking \"\*.dat\"" track.log
+
+  # this string announces to server that we want a test that
+  # interrupts the transfer when started from 0 to cause resume
+  contents="status-batch-retry"
```

Comment:
Thanks for documenting this. When I first explored these tests, I was often confused by why these magic strings caused certain things to happen; this is really helpful for similar newcomers.

Reply:
I actually need to revise this, since this comment is pulled in from another test assertion. But I will replace it with a hopefully equally helpful comment.